lib/Settings/UserFlagsController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace MailPoet\Settings;
+
+use MailPoet\Models\UserFlag;
+use MailPoet\WP\Functions as WPFunctions;
+
+class UserFlagsController {
+
+  /**
+   * @var array|null
+   */
+  private $data = null;
+  
+  /**
+   * @var array
+   */
+  private $defaults;
+
+  function __construct() {
+    $this->defaults = [
+      'last_announcement_seen' => false,
+      'editor_tutorial_seen' => false,
+    ];
+  }
+
+  function get($name) {
+    $this->ensureLoaded();
+    if (empty($this->data[$name])) {
```

Comment:
I don't think this works as intended. If we set a value to `false` or `''` as we always do, this will return the default value. For now, there is no difference because all defaults are `false` but in the future, this may be different. The default value could be `true` and this would break.